docs/Gemfile.lock

```diff
@@ -3,26 +3,27 @@ GEM
   specs:
     RedCloth (4.2.9)
     blankslate (2.1.2.4)
-    celluloid (0.15.2)
-      timers (~> 1.1.0)
-    classifier (1.3.4)
-      fast-stemmer (>= 1.0.0)
-    coffee-script (2.3.0)
+    celluloid (0.16.0)
+      timers (~> 4.0.0)
+    classifier-reborn (2.0.3)
+      fast-stemmer (~> 1.0)
+    coffee-script (2.4.1)
       coffee-script-source
       execjs
-    coffee-script-source (1.7.1)
+    coffee-script-source (1.9.1.1)
     colorator (0.1)
-    execjs (2.2.1)
+    execjs (2.5.2)
     fast-stemmer (1.0.2)
-    ffi (1.9.3)
-    jekyll (2.2.0)
```

Comment:
It seems like you also update a bunch of other tools besides Jekyll here. Are those supposed to be part of this PR? (`celluloid`, `classifier`, `coffee-script`, `coffee-script-source`, `execjs`, `ffi`)